spec/openapi.yaml

```diff
@@ -6468,6 +6468,68 @@ components:
             The subject field is used to denote subject of the MMS message and
             has a maximum size of 64 characters long
           example: New Products
+    DetailMessage:
```

Comment:
Think you might want to take another pass at this @SamRussellMM. Just doing a quick check, it looks like `destination_address_country`, `account_id`, `units`, and `status_description` are missing.

Reply:
Cheers Liam, have added the missing fields. Noticed they weren't highlighted or mentioned in the excel spreadsheet, so I didn't include them the first time.